internal/tag_keys/tag_keys.go

```diff
@@ -1,8 +1,9 @@
 package tag_keys
 
 import (
-	"github.com/hashicorp/hcl/v2/hclwrite"
 	"strings"
+
+	"github.com/hashicorp/hcl/v2/hclwrite"
```

Comment:
what's the change here?

Reply:
automattic golang formatting... nothing changed... I guess the formatter fixed it this way...